design_drafts/components_architecture_and_urdf_examples.md

```diff
@@ -63,37 +63,25 @@ Note:
   * Examples: KUKA FRI, ABB Yummy, Schunk LWA4p, etc.
 
 ```xml
-  <ros2_control name="2DOF_System_Robot_MultiInterface" type="system">
+  <ros2_control name="2DOFSystemRobotMultiInterface" type="system">
     <hardware>
-      <classType>ros2_control_demo_hardware/2DOF_System_Hardware_MultiInterface</classType>
+      <classType>ros2_control_demo_hardware/2DOFSystemHardwareMultiInterface</classType>
       <param name="example_param_write_for_sec">2</param>
       <param name="example_param_read_for_sec">2</param>
     </hardware>
     <joint name="joint1">
       <classType>ros2_control_components/MultiInterfaceJoint</classType>
-      <commandInterfaceType name="position">
-        <param name="min_value">-1</param>
-        <param name="max_value">1</param>
-      </commandInterfaceType>
-      <commandInterfaceType name="velocity">
-        <param name="min_value">-1</param>
-        <param name="max_value">1</param>
-      </commandInterfaceType>
-      <commandInterfaceType name="effort">
-        <param name="min_value">-0.5</param>
-        <param name="max_value">0.5</param>
-      </commandInterfaceType>
+      <commandInterfaceType name="position" min="-1" max="1"/>
```

Comment:
I'd always consider min-max to be optional

Reply:
I am not sure if this is not confusing for the users. Because in some cases is attribute and in some case is child element. I understand why and when, but a new user could have issues with it.